megamek/src/megamek/client/ui/swing/PlayerListDialog.java

```diff
@@ -19,42 +19,88 @@
 import megamek.client.ui.swing.util.UIUtil;
 import megamek.common.Player;
 import megamek.common.Team;
+import megamek.common.event.GameListener;
+import megamek.common.event.GameListenerAdapter;
+import megamek.common.event.GamePhaseChangeEvent;
 import megamek.common.preference.IPreferenceChangeListener;
 import megamek.common.preference.PreferenceChangeEvent;
 
 import javax.swing.*;
 import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.text.MessageFormat;
 
-public class PlayerListDialog extends JDialog implements IPreferenceChangeListener {
+public class PlayerListDialog extends JDialog implements ActionListener, IPreferenceChangeListener {
 
     private static final long serialVersionUID = 7270469195373150106L;
 
     private JList<String> playerList = new JList<>(new DefaultListModel<>());
 
     private Client client;
-
-    public PlayerListDialog(JFrame parent, Client client) {
-        super(parent, Messages.getString("PlayerListDialog.title"), false);
+    private JButton butOkay;
+    private boolean model;
+
+    private static final String MSG_OKAY = Messages.getString("Okay");
+    private static final String MSG_TITLE = Messages.getString("PlayerListDialog.title");
```

Comment:
You removed many of those static final localized strings in the other PR :) just saying, I don't mind

Reply:
There is no need to change this, I just noticed it.